### PR TITLE
DOC-3321 Added upgrade notes and skipSizeQuery to config table

### PR DESCRIPTION
**Pull request by benpesterr3**

### CODE REVIEW

File: content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md [hunk 1]
```diff
@@ -43,9 +38,19 @@ As the name suggests, this is a collaborative method for recovering data. For ma
...
-* Your Business Network disaster recovery policy
-* The Corda platform requirements
-* How and when the **LedgerSync** and **LedgerRecover** CorDapps should be used
+* Your Business Network disaster recovery policy.
+* The Corda platform requirements.
+* How and when the **LedgerSync** and **LedgerRecover** CorDapps should be used.
+
+## In this version
+
+You can now use the `skipSizeQuery` configuration in the Ledger Recover CorDapp in cases where you cannot grant `SELECT` permission to PostGres admin to check the file size of the exported transactions before exporting them. This check normally verifies that the size of files to be exported does not exceed your node’s disk size.
```
Comment (alexandradh) [inline content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md:47]:
```suggestion
You can now use the `skipSizeQuery` configuration in the Ledger Recover CorDapp in cases where you cannot grant `SELECT` permission to PostGres admin to check the size of artifacts (transactions, attachments, network parameters). In Automatic Ledger Recover, this check normally verifies that the size of transactions (including their backchain with attachments and network parameters) does not exceed the maximum allowed message size. In Manual Ledger Recover, this check normally verifies that the size of files to be exported does not exceed your node’s available disk size. In Manual Ledger Recover, 
```

File: content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md [hunk 2]
```diff
@@ -43,9 +38,19 @@ As the name suggests, this is a collaborative method for recovering data. For ma
...
-* Your Business Network disaster recovery policy
-* The Corda platform requirements
-* How and when the **LedgerSync** and **LedgerRecover** CorDapps should be used
+* Your Business Network disaster recovery policy.
+* The Corda platform requirements.
+* How and when the **LedgerSync** and **LedgerRecover** CorDapps should be used.
+
+## In this version
+
+You can now use the `skipSizeQuery` configuration in the Ledger Recover CorDapp in cases where you cannot grant `SELECT` permission by the PostGres admin to check the file size of the exported transactions before exporting them. This check normally verifies that the size of files to be exported does not exceed your node’s disk size.
```
Comment (ivanterziev-r3) [inline content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md:47]: A minor thing but should "Ledger Recover CorDapp" be `LedgerRecover` (as per https://docs.r3.com/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.html#ledgerrecover)?

Comment (ivanterziev-r3) [reply content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md:47]: Also: PostGres > PostgreSQL

File: content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md [hunk 3]
```diff
@@ -43,9 +38,19 @@ As the name suggests, this is a collaborative method for recovering data. For ma
...
+
+This change does not remove the file size limit from a network, so if the exported transactions do exceed your node's disk size, the recovery operation can still fail. In this case, you will get a generic error from Corda, rather than a specific reason for the recovery failure. 
+
+### Changes from V1.2
+
+In Collaborative Recovery V1.2, LedgerSync has been modified to be compatible with the [Archive Service](../archiving/archiving-setup).
```
Comment (ivanterziev-r3) [inline content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md:53]: Does this link work?

File: content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md [hunk 4]
```diff
@@ -43,13 +38,23 @@ As the name suggests, this is a collaborative method for recovering data. For ma
...
+
+## In this version
+
+You can now use the `skipSizeQuery` configuration in `LedgerRecover` in cases where you cannot grant `SELECT` permission by the PostgreSQL admin to check the file size of the exported transactions before exporting them. This check normally verifies that the size of files to be exported does not exceed your node’s disk size.
+
+This change does not remove the file size limit from a network, so if the exported transactions do exceed your node's disk size, the recovery operation can still fail. In this case, you will get a generic error from Corda, rather than a specific reason for the recovery failure.
+
+### Changes in V1.2
+
+In Collaborative Recovery V1.2, LedgerSync has been modified to be compatible with the [Archive Service](/en/platform/corda/4.8/enterprise/node/archiving/archiving-setup.md).
```
Comment (ivanterziev-r3) [inline content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md:53]:
```suggestion
In Collaborative Recovery V1.2, LedgerSync has been modified to be compatible with the [Archive Service](../../../../../../../en/platform/corda/4.8/enterprise/node/archiving/archiving-setup.md).
```

File: content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md [hunk 5]
```diff
@@ -22,12 +22,7 @@ weight: 100
 * Business Network Operators (BNOs)
 * Corda developers
 
-
-## Introducing Collaborative Recovery V1.2
-
-In Collaborative Recovery V1.2, LedgerSync has been modified to be compatible with the [Archive Service](../archiving/archiving-setup).
-
-### In Collaborative Recovery V1.1
+## Introducing Collaborative Recovery V1.2.1
```
Comment (ivanterziev-r3) [inline content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md:25]:
```suggestion

## Introducing Collaborative Recovery V1.2.1
```

**Pull request merged**